docker/dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM 267292272963.dkr.ecr.us-east-1.amazonaws.com/uptycs/alpine:v3
```

Comment:
move this to internal docker repo ? .. uptycs/docker (it will git pull code from this public repo)